app/client/cli/account.go

```diff
@@ -0,0 +1,72 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/utility/types"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	actorCmd := NewAccountCommand()
+	actorCmd.Flags().StringVar(&pwd, "pwd", "", "passphrase used by the cmd, non empty usage bypass interactive prompt")
+	rootCmd.AddCommand(actorCmd)
+}
+
+func NewAccountCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "Account",
+		Short:   "Account specific commands",
+		Aliases: []string{"account"},
+		Args:    cobra.ExactArgs(0),
+	}
+
+	cmd.AddCommand(accountCommands()...)
+
+	return cmd
+}
+
+func accountCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "Send <fromAddr> <to> <amount>",
+			Short:   "Send <fromAddr> <to> <amount>",
+			Long:    "Sends <amount> to address <to> from address <fromAddr>",
+			Aliases: []string{"send"},
+			Args:    cobra.ExactArgs(3),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				// TODO(#150): update when we have keybase
+				pk, err := readEd25519PrivateKeyFromFile(privateKeyFilePath)
+				if err != nil {
+					return err
+				}
+				// currently ignored since we are using the address from the PrivateKey
```

Comment:
Can you add some detail about the path forward on this comment?

Reply:
Please let me know if this is OK and/or enough https://github.com/pokt-network/pocket/pull/177/commits/e91a1ab9952a073d4d8b18ea4776350b3d7da0be